dev-vcs/subversion/subversion-1.14.3.recipe

```diff
@@ -31,82 +32,88 @@ PROVIDES="
 	cmd:svnserve = $portVersion compat >= 1.8
 	cmd:svnsync = $portVersion compat >= 1.8
 	cmd:svnversion = $portVersion compat >= 1.8
-	lib:libsvn_client_1 = 0.0.0 compat >= 0
-	lib:libsvn_delta_1 = 0.0.0 compat >= 0
-	lib:libsvn_diff_1 = 0.0.0 compat >= 0
-	lib:libsvn_fs_1 = 0.0.0 compat >= 0
-	lib:libsvn_fs_fs_1 = 0.0.0 compat >= 0
-	lib:libsvn_fs_util_1 = 0.0.0 compat >= 0
-	lib:libsvn_fs_x_1 = 0.0.0 compat >= 0
-	lib:libsvn_ra_1 = 0.0.0 compat >= 0
-	lib:libsvn_ra_local_1 = 0.0.0 compat >= 0
-	lib:libsvn_ra_serf_1 = 0.0.0 compat >= 0
-	lib:libsvn_ra_svn_1 = 0.0.0 compat >= 0
-	lib:libsvn_repos_1 = 0.0.0 compat >= 0
-	lib:libsvn_subr_1 = 0.0.0 compat >= 0
-	lib:libsvn_swig_perl_1 = 0.0.0 compat >= 0
-	lib:libsvn_wc_1 = 0.0.0 compat >= 0
+	lib:libsvn_client_1$secondaryArchSuffix = 0.0.0 compat >= 0
+	lib:libsvn_delta_1$secondaryArchSuffix = 0.0.0 compat >= 0
+	lib:libsvn_diff_1$secondaryArchSuffix = 0.0.0 compat >= 0
+	lib:libsvn_fs_1$secondaryArchSuffix = 0.0.0 compat >= 0
+	lib:libsvn_fs_fs_1$secondaryArchSuffix = 0.0.0 compat >= 0
+	lib:libsvn_fs_util_1$secondaryArchSuffix = 0.0.0 compat >= 0
+	lib:libsvn_fs_x_1$secondaryArchSuffix = 0.0.0 compat >= 0
+	lib:libsvn_ra_1$secondaryArchSuffix = 0.0.0 compat >= 0
+	lib:libsvn_ra_local_1$secondaryArchSuffix = 0.0.0 compat >= 0
+	lib:libsvn_ra_serf_1$secondaryArchSuffix = 0.0.0 compat >= 0
+	lib:libsvn_ra_svn_1$secondaryArchSuffix = 0.0.0 compat >= 0
+	lib:libsvn_repos_1$secondaryArchSuffix = 0.0.0 compat >= 0
+	lib:libsvn_subr_1$secondaryArchSuffix = 0.0.0 compat >= 0
+	lib:libsvn_swig_perl_1$secondaryArchSuffix = 0.0.0 compat >= 0
+	lib:libsvn_wc_1$secondaryArchSuffix = 0.0.0 compat >= 0
 	"
 REQUIRES="
-	haiku
-	lib:libapr_1
-	lib:libaprutil_1
-	lib:libcrypto
-	lib:libexpat
-	lib:libiconv
-	lib:liblz4
-	lib:libserf_1
-	lib:libsqlite3
-	lib:libssl
-	lib:libxml2
-	lib:libz
+	haiku$secondaryArchSuffix
+	lib:libapr_1$secondaryArchSuffix
+	lib:libaprutil_1$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libexpat$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:liblz4$secondaryArchSuffix
+	lib:libserf_1$secondaryArchSuffix
+	lib:libsqlite3$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
```

Comment:
As this contains perl modules in the vendor_perl directory, should we maybe add `vendor_perl` to the requirements here?

Although then we should probably split the perl bindings into a subpackage (similar to git) to avoid a hard perl dependency of the main package.